packages/ui/src/pages/Settings/Settings.tsx

```diff
@@ -50,7 +57,14 @@ const Settings = () => {
         expanded={expanded === ACCORDION_WALLET_CONNECT}
         onChange={() => onToggle(ACCORDION_WALLET_CONNECT)}
       >
-        <AccordionSummaryStyled expandIcon={<ExpandMoreIcon size={20} />}>
+        <AccordionSummaryStyled
+          expandIcon={<ExpandMoreIcon size={20} />}
+          data-cy={
+            expanded === ACCORDION_WALLET_CONNECT
+              ? 'button-collapse-wallet-connect'
+              : 'button-expand-wallet-connect'
+          }
```

Comment:
This is too much specific code for the tests IMHO, and it's repeated.
it'd be better to have something simple
```suggestion
          data-cy="accordion-title-wallet-connect"
```

and then check for specific attributes. In this case there's a good one with `aria-expanded`

```
  .should('have.attr', 'aria-expanded', true);
```

Reply:
Updated to use the simpler tag 👍

I'll just click in this test and check for the accordion default state in a wallet-connect test rather than here as we need to add one there for default connected / default non-connected (non watch mode). Makes more sense to be in a separate feature test rather than be part of the checks here.